src/ort_genai_c.cpp

```diff
@@ -531,6 +531,39 @@ size_t OGA_API_CALL OgaStringArrayGetCount(const OgaStringArray* string_array) {
   return reinterpret_cast<const std::vector<std::string>*>(string_array)->size();
 }
 
+OgaResult* OgaCreateAdapters(const OgaModel* model, OgaAdapters** adapters_p) {
```

Comment:
Should `adapters_p` be `out` for consistency with other methods?

Reply:
Will address this also in a separate PR so I can unblock the C# API development. Will get to it soon.